npm/ng-packs/packages/core/src/lib/services/local-storage-listener.service.ts

```diff
@@ -1,18 +1,16 @@
 import { DOCUMENT } from '@angular/common';
 import { Injectable, inject } from '@angular/core';
-import { OAuthService } from 'angular-oauth2-oidc';
 
 @Injectable({
   providedIn: 'root',
 })
 export class LocalStorageListenerService {
   protected readonly window = inject(DOCUMENT).defaultView;
-  protected readonly authService = inject(OAuthService);
 
   constructor() {
     this.window.addEventListener('storage', event => {
       if (event.key === 'access_token' && event.newValue === null) {
-        this.authService.logOut();
+        this.window.location.reload();
```

Comment:
We've AuthService in core package try to use it please

Reply:
I have tried it doesn't work as expected.

Reloading other tabs is giving us what we want to achieve. So no need to use `AuthService`